www/src/theme.js

```diff
@@ -24,33 +24,30 @@ const colors = {
   darker: '#20232a', // really dark blue
   brand: '#61dafb', // electric blue
   brandLight: '#bbeffd',
+  brandOnWhite: '#10819b',
   text: '#1a1a1a', // very dark grey / black substitute
-  subtle: '#777', // light grey for text
+  subtle: '#6d6d6d', // light grey for text
+  subtleOnDark: '#999',
```

Comment:
nit: `subtleDark` ?

Reply:
I think subtleOnDark is nicely verbose in this instance, showing it's for use on dark backgrounds. Would you agree?

Reply:
Sure that's okay 😄  Feel free to ignore "nit" comments